### ISSUE TITLE
Icon: create white version of all background icons

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#593]
To support windows white-on-black high contrast mode we need to create white (or very light grey) versions of all our icons.

**Comment by ianmcburnie** [comment]
Closing this as we plan to deprecate background icon usage in v10.
